### DISCUSSION TITLE
Need help to explain this diff view structure

### DISCUSSION THREAD

**Discussion by IOLOII** [alefragnani/vscode-project-manager#748] [Q&A] [answered]
When open this extension, sometimes in the new profiles (maybe first install) will get left structure in up pictures, but in the normal  to use this extension was right structure, I didn't find somewhere to change this view structure

<img width="748" alt="image" src="https://github.com/alefragnani/vscode-project-manager/assets/34856171/e527f045-814a-431b-a176-6be94c1d0570">

**Comment by alefragnani** [comment, marked as answer]
Hi @IOLOII ,

I guess this may be happenng because of the _File Icon Theme_ you have selected for some workspace and/or Profile, But is should be temporary, until the extension is fully loaded.

I see this misalignment when switching to a File Icon Theme that has folder icons, for instance, from _None_ or _Minimal_ to _Seti_. But once you effetively select the File Icon Theme, the icons are properly aligned in the view.

Other than that, I don't see why this would happend, unfortunately.

Hope this helps

**Comment by IOLOII** [comment]
Thank you.

I tried to do as you said, but don't happend again.
It just sometime appear in my vscode. Haaaaa

from 阿里邮箱 macOS------------------------------------------------------------------
发件人：Alessandro ***@***.***>
日　期：2024年07月09日 08:14:16
***@***.***>
***@***.***>; ***@***.***>
主　题：Re: [alefragnani/vscode-project-manager] Need help to explain this diff view structure (Discussion #748)

Hi @IOLOII ,
I guess this may be happenng because of the File Icon Theme you have selected for some workspace and/or Profile, But is should be temporary, until the extension is fully loaded.
I see this misalignment when switching to a File Icon Theme that has folder icons, for instance, from None or Minimal to Seti. But once you effetively select the File Icon Theme, the icons are properly aligned in the view.
Other than that, I don't see why this would happend, unfortunately.
Hope this helps
